modules/audio.md

```diff
@@ -0,0 +1,124 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+
+## <img class="constrain-sm" src="https://s3.amazonaws.com/technicalmachine-assets/technical-io/modules/usb.png"> Audio
+
+[<i class="fa fa-github">View source on Github</i>](https://github.com/tessel/tessel-av)
+
+### Step 1
+
+Make a directory inside your "tessel-code" folder called "audio", change directory into that folder, and initialize a tessel project:
+
+`mkdir audio; cd audio; t2 init`
+
+Download [this mp3](https://dl.dropboxusercontent.com/u/3531958/yoda-mudhole.mp3) into the "audio" directory: 
+
+`wget https://dl.dropboxusercontent.com/u/3531958/yoda-mudhole.mp3`
```

Comment:
It's important to keep this tutorial accessible to new coders. It's also important to make it work across operating systems. That leads me to two recommendations:

1. Create a new step explaining what we're doing by running these commands- namely, setting up the folder (and why?)
2. Lean away from prescribing commands like `wget` because not all users know what this is or have it